### PR TITLE
Update to recommendations made by documentation team review

### PR DESCRIPTION
**Pull request by lmzuccarelli**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,14 +44,14 @@ This process refers to building the operator in a way that it can be installed l
...
-2. Build and push the image index for operator-registry:
+2. Build and push the image index for `operator-registry`:
    ```sh
    $ export INDEX_IMG=<registry>/<username>/external-dns-operator-bundle-index:1.0.0
    $ make index-image-build
    $ make index-image-push
    ```
 
-3. Create the catalogsource (registry secret may need to be linked to the external-dns-operator's pod created in `openshift-marketplace`):
+3. Create the `Catalogsource` object (you may need to link the registry secret to the pod of `external-dns-operator` created in `openshift-marketplace`):
```
Comment (chinmayi-chandrasekar) [inline README.md:54]: Is this - (you may need to link the registry secret to the pod of `external-dns-operator` created in `openshift-marketplace`)  optional for the developer? and is openshift-marketplace the namespace? If yes, where is it created as part of the procedure?

File: README.md [hunk 2]
```diff
@@ -1,35 +1,41 @@
...
+1. Build and push the Operator image to a registry:
    ```sh
    $ export IMG=<registry>/<username>/external-dns-operator:latest
    $ make image-build
    $ make image-push
    ```
-2. Run `make deploy` to deploy the operator
-3. Now you can deploy an instance of ExternalDNS:
-    * Run the following command to create the credentials secret for AWS:
+2. Run the following command to deploy the `ExternalDNS`  Operator:
```
Comment (chinmayi-chandrasekar) [inline README.md:17]: Please remove one space between `ExternalDNS` and Operator.

File: README.md [hunk 3]
```diff
@@ -1,35 +1,41 @@
...
+    ```
+    $ make deploy
+    ```
+3. Deploy an instance of ExternalDNS:
+    * Create the `credentials`  secret for AWS:
```
[the remark this answers is not in the document]
Comment (lmzuccarelli) [reply README.md:22]: fixed

File: README.md (hunk 2 above)
Comment (lmzuccarelli) [reply README.md:17]: fixed

File: README.md [hunk 4]
```diff
@@ -1,35 +1,41 @@
...
+       *Note*: For other providers, see options listed in `api/v1alpha1 /externaldns_types.go`.
```
[the remark this answers is not in the document]
Comment (lmzuccarelli) [reply README.md:23]: removed - fixed

File: README.md [hunk 5]
```diff
@@ -78,4 +84,4 @@ This process refers to building the operator in a way that it can be installed l
       sourceNamespace: openshift-marketplace
     EOF
     ```
-    **Note**: Same thing can be done via the console: `Operators` -> `OperatorHub`, search for `ExternalDNS operator` and install the operator.
+    **Note**: You can install the `ExternalDNS` Operator through web console: Navigate to  `Operators` -> `OperatorHub`, search for `ExternalDNS operator`,  and install the operator.
```
[the remark this answers is not in the document]
Comment (lmzuccarelli) [reply README.md:87]: fixed

File: README.md [hunk 6]
```diff
@@ -1,35 +1,41 @@
 # ExternalDNS Operator
 
-The ExternalDNS Operator deploys and manages [ExternalDNS](https://github.com/kubernetes-sigs/external-dns), which dynamically manages
-external DNS records in specific DNS Providers for specific Kubernetes resources.
-
-This Operator is in the early stages of implementation. For the time being, please reference the
+The `ExternalDNS` Operator allows you to deploy and manage [ExternalDNS](https://github.com/kubernetes-sigs/external-dns), a cluster-internal DNS server in Kubernetes which makes Kubernetes resources discoverable through public DNS servers. \
```
[the remark this answers is not in the document]
Comment (alebedev87) [reply README.md:3]: I'd say `a cluster-internal component which makes ...`

**Pull request merged**